### ISSUE TITLE
[KButton]: Add aria-haspopup and aria-expanded to KButton when it has a KDropdownMenu related

### ISSUE THREAD

**Issue by AlexVelezLl** [learningequality/kolibri-design-system#832]
🌱 Are you new to the codebase? Welcome! Please see [the contributing guidelines](https://github.com/learningequality/kolibri-design-system/blob/develop/CONTRIBUTING.md).

## Product
Kolibri, Studio, KDS, etc.

## Expected behavior

KButton should have the required [aria-haspopup](https://w3c.github.io/aria/#aria-haspopup) property, and the [aria-expanded](https://w3c.github.io/aria/#aria-expanded) state when it activates a dropdown menu.

The rendered button should have aria-haspopup="menu" if it has a menu slot defined, aria-expanded="false" if the dropdown is closed, and aria-expanded="true" if the dropdown is opened.

## Actual behavior
We dont have any of these properties in kbutton right now.

**Comment by josephinoo** [comment]
Hi @AlexVelezLl , can you assign this issue to me ?

**Comment by AlexVelezLl** [comment]
Hi @josephinoo! For sure! Thanks for your interest in contributing to Kolibrí 🤗 . I will assign this issue to you. Let us know if you have any questions :)

**Comment by MisRob** [comment]
Hi @josephinoo, I wanted to mention that [Learning Equality will be closed from December 23 to January 5](https://github.com/learningequality/kolibri/discussions/12956).
